.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,32 @@
+name: Report a bug
+description: Something is not working as expected? Tell us the problem
+labels: ["kind/bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for using Pelorus and reporting us back this problem, this helps
+        the project always improve!
+  - type: dropdown
+    attributes:
+      label: OpenShift version
+      description: In what version of OpenShift the problem ocurred?
+      options:
+        - 4.10
+        - 4.11
+        - 4.12
+        - 4.13
+        - Not related to OpenShift
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Problem description
+      description: |
+        Tell us what happened. The more info you provide, the faster the
```

Comment:
Can we have another few:

```
Steps to reproduce:
[What steps were performed to reproduce the bug]
```

```
Expected behavior:
[Description of what the user expects to happen when performing the steps to reproduce]
```

```
Actual behavior:
[Description of what actually happens when performing the steps to reproduce]

```

Reply:
steps to reproduce I think is good, the other 2 I feel user can not understand. Do you want really want to add them?

Reply:
Examples:
Actual behavior:
 > I can see 5 deployments of commit time exporter with provided config.

Expected behavior:
> I should see 1 deployment of commit time exporter with provided config


It's pretty standard thing across different bug systems. This helps to check what the user was expecting vs what actually happened.